group_vars/all.yml

```diff
@@ -44,3 +44,7 @@ enable_maintenance_mode_source: "FALSE"
 # Set disable_maintenance_mode_dest to TRUE initially, to avoid undefined
 # variable errors. It can be overridden in migration_vars.yml
 disable_maintenance_mode_dest: "FALSE"
+
+# This is a little-used variable; only used when there are file permissions
+# issues from files that were not created/saved by Drupal.
+chmod: "FALSE"
```

Comment:
I'd say we would always want to do this. It doesn't hurt and would save us having to discover which sites have this issue.

Reply:
Hm. I wonder how much time it takes. However, we're done with the mass migrations, where seconds count, so it's probably worth it here.